libs/blocks/hero-marquee/hero-marquee.js

```diff
@@ -60,22 +85,25 @@ function decorateBg(el) {
   el.remove();
 }
 
+function wrapInnerHTMLInPTag(el) {
+  const innerDiv = el.querySelector(':scope > div');
+  const containsPTag = [...innerDiv.childNodes].some((node) => node.nodeName === 'P');
+  if (!containsPTag) innerDiv.innerHTML = `<p>${innerDiv.innerHTML}</p>`;
```

Comment:
This looks like something that Kodiak might sniff. Could we use another strategy (not `innerHTML`)? Iterating through nodes or other approaches you think are best.

Reply:
Good call. On this we just need the innerDiv.firstChild etc. and can avoid using innerHTML etc. This has been upated